src/groups/mqb/mqbblp/mqbblp_queue.cpp

```diff
@@ -766,8 +766,8 @@ void Queue::onPushMessage(
     mqbi::DispatcherEvent* dispEvent = dispatcher()->getEvent(this);
 
     (*dispEvent)
-        .setType(mqbi::DispatcherEventType::e_PUSH)
         .setSource(this)
+        .makePushEvent()
         .setBlob(appData)
         .setOptions(options)
         .setGuid(msgGUID)
```

Comment:
Note that initially we are building `DispatcherEvent`, and after `makePushEvent()` we are calling setters for `DispatcherPushEvent`. So we switch between objects amidst these setters calls. This interface is not perfect, but I have plans to replace it by calling constructors with all args instead.

So instead of this
```
*dispEvent)
        .setSource(this)
        .makePushEvent()
        .setBlob(appData)
        .setOptions(options)
        .setGuid(msgGUID)
```

We will have smth like this:
```
*dispEvent)
        .setSource(this)
        .makePushEvent(appData, options, msgGUID, ...)
```

I plan to do it in a separate PR, with performance benchmarks, to limit the number of changes in this PR.